seqr/views/apis/saved_variant_api_tests.py

```diff
@@ -155,10 +154,8 @@ def test_saved_variant_data(self):
             'genesById', 'locusListsByGuid',
         })
         variants = response_json['savedVariantsByGuid']
-        self.assertSetEqual(
-            set(variants.keys()),
-            {'SV0000002_1248367227_r0390_100', 'SV0000001_2103343353_r0390_100', COMPOUND_HET_1_GUID, COMPOUND_HET_2_GUID}
-        )
+        guid_set = {'SV0000002_1248367227_r0390_100', 'SV0000001_2103343353_r0390_100', COMPOUND_HET_1_GUID, COMPOUND_HET_2_GUID}
+        self.assertSetEqual({guid for guid in variants.keys() if guid in guid_set}, guid_set)
```

Comment:
This doesn't test anything. It says "assert that the subset of this set that is in `guid_set` equals `guid_set`, which is obviously true but meaningless. Please add the new saved variant object to a different family to prevent this issue

Reply:
When the set(variants.key()) misses any elements in guid_set, then it will be an assertion error.  The test asserts that the guid_set is a subset of set(variants.keys()), or set(variants.keys()) contains guid_set. It is not always true.
The issue of using a different family is that I have to create a complete set of saved_variants, variant_tags, variant_tag_type (maybe), samples, individuals, and a family. I think they are too many. How do you think about that?

Reply:
Oh sorry I misread this code. The much clearer way to write this is `set(variants.keys())`